src/layers/MapboxLayer.js

```diff
@@ -122,6 +122,8 @@ export default class MapboxLayer extends Layer {
       pitchWithRotate: false,
       scrollZoom: false,
       touchZoomRotate: false,
+      fadeDuration:
+        'fadeDuration' in this.options ? this.options.fadeDuration : 300,
```

Comment:
I guess mapbox already makes this test for you:
```suggestion
      fadeDuration: this.options.fadeDuration,
```

Reply:
Would be nice but I don't see any labels at all if fadeDuration is undefined

Reply:
so sad .